charts/management-portal/templates/deployment.yaml

```diff
@@ -49,6 +49,7 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{ $https := ternary "http" "https" (or .Values.disable_tls (not .Values.ingress.tls)) }}
```

Comment:
Let's put this on top of the file, first few lines

Reply:
Done!